test/stripe/order_return_test.rb

```diff
@@ -0,0 +1,24 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class OrderReturnTest < Test::Unit::TestCase
+    should "return_order should be listable" do
+      @mock.expects(:get).once.returns(make_response(make_order_return_array))
+      returns = Stripe::OrderReturn.list
+      assert returns.data.kind_of?(Array)
+      returns.each do |ret|
+        assert ret.kind_of?(Stripe::OrderReturn)
+      end
+    end
+
+    should "returns should not be updateable" do
+      assert_raises NoMethodError do
```

Comment:
Yeah, that's not entirely unreasonable. I think the server-side check would just give you a 404 so I'm not sure which one is preferable. Looks like we have a few instances of asserting this behavior by checking that it raises `NoMethodError`, but I'm happy to take it out (though the behavior obviously won't change)